R/load.project.R

```diff
@@ -275,7 +275,25 @@ load.project <- function(override.config = NULL)
   config <- .normalize.config(config,
                               setdiff(names(default.config), c("version", "libraries", "logging_level")),
                               .boolean.cfg)
-
+  
+  # add some project config if it exists in a file called custom.dcf
+  custom_cfg <- file.path('config', 'custom.dcf') 
+  if (file.exists(custom_cfg)) {
+        custom_cfg <- translate.dcf(custom_cfg)
+ 
+        # all items are char, so convert them if they look like logical or numeric
+        convert_type <- function(v){
```

Comment:
Yes, I looked at the .boolean.cfg and it did look very similar.  I agree it would be more re-usable to follow the paradigm, rather than the messy inline function I used here.  I'll tidy up in my branch and put out again for review.